pkgs/development/libraries/botan/generic.nix

```diff
@@ -46,7 +46,7 @@ stdenv.mkDerivation rec {
     inherit version;
     description = "Cryptographic algorithms library";
     maintainers = with maintainers; [ raskin ];
-    platforms = ["x86_64-linux" "i686-linux" "x86_64-darwin"];
+    platforms = ["aarch64-linux" "x86_64-linux" "i686-linux" "x86_64-darwin"];
```

Comment:
```suggestion
    platforms = [ platforms.linux "x86_64-darwin"];
```

Reply:
That gives a list in a list, I guess you'd want something like:

```
platforms = platforms.linux ++ platforms.darwin;
```

(Or `x86_64-darwin` specifically if you believe it wouldn't build on macOS AArch64.).

Of course, you may as well use `platforms.all` then as well.

Reply:
It seems like `platforms.unix` was the previous value, but the explicit list has been used instead, because it would not build under aarch64. As it now does i don't see any reason to exclude anything more, than we must, so i would like to remove the explicit list.

Would the old value of `platforms.unix` be apropiate here, or should `platforms.all` be used here? `platforms.linux ++ platforms.darwin` seems a bit arbitrary to me.

Reply:
`platforms.unix` sounds sensible then :+1: